libcloudforensics/providers/azure/internal/account.py

```diff
@@ -42,7 +42,7 @@ class AZAccount:
   """
 
   def __init__(self,
-               default_resource_group_name: str,
+               default_resource_group_name: str = '',
```

Comment:
Can you update the docstring to reflect that this parameter is now Optional?

Reply:
done